.github/workflows/test.yml

```diff
@@ -87,6 +89,6 @@ jobs:
         run: NUMBER_OF_THRASH_ITER=20 FILL_PERCENT_CAPACITY=20 ./hack/e2e-test.sh --number-of-compute $NUMBER_OF_COMPUTE_NODES
 
       - name: Execute golang based E2E tests
-        run: pushd e2e; go test -v . ; popd
+        run: pushd e2e; go test -v . -timeout 30m; popd
```

Comment:
are you actually reading this anywhere ? From what I understood in the code below, you're just using the `defaultReconcilerTimeout`, which is 30.

Reply:
my bad. 

I agree, 30 minutes can be too optimistic. I would raise it to 60.

Reply:
Lol I have since upped it to 2 hours. But eventually we should make it as short as possible, of course.